libs/blocks/tag-selector/tag-selector.css

```diff
@@ -76,3 +76,36 @@
   background: transparent;
   border: none;
 }
+
+/* SPINNER */
```

Comment:
Probably don't need this comment since the class name is spinner.

Reply:
Removed